data/supporting_members.yaml

```diff
@@ -526,3 +525,195 @@ data:
+        - icon: twitter
+          url: https://twitter.com/NielsDeGraef
+    - name: Wout Schellaert
+      nickname: stro
+      photo: https://zeus.ugent.be/zeuswpi/1gMbZvu6.jpg
+      quote: \"
+      description: Blues, Rust, en liefde
+      links:
+        - icon: twitter
+          url: https://twitter.com/WoutSchellaert
+        - icon: github
+          url: https://github.com/wschella/
+        - icon: linkedin
+          url: https://www.linkedin.com/in/woutschellaert/
+        - url: http://schellaert.org/
+    - name: Francis Begyn
+      nickname: thecy
+      photo: https://zeus.ugent.be/zeuswpi/Xl5-9fld.jpg
+      quote: Ik wil eigenlijk nog gaan feesten, maar flask migrations zijn fucked
+      description: Ik breek dingen die werken en fix dingen die gebroken zijn.
+      links:
+        - icon: facebook
+          url: https://www.facebook.com/begynfrancis
+        - icon: twitter
+          url: https://twitter.com/fbegyn
+        - icon: linkedin
+          url: https://www.linkedin.com/in/fbegyn/
+        - url: https://francis.begyn.be
+    - name: Felix Van der Jeugt
+      nickname: noctua
+      photo: https://zeus.ugent.be/zeuswpi/9_L03Pm8.jpg
+      quote: |-
+        <noctua> dwergen
+        <noctua> dwergen
+        <noctua> dwergen
+        <noctua> dwergen
+        <noctua> Sorry guys
+        <noctua> dwergen
+        <noctua> dwergen
+      description: Komt geen Ice Tea's meer halen in de kelder :(
+      links:
+        - icon: github
+          url: https://www.github.com/ninewise
+        - url: https://users.ugent.be/~fvdrjeug/
+    - name: Rien Maertens
+      nickname: met stokjes
+      photo: https://zeus.ugent.be/zeuswpi/udVo8mPM.jpg
+      quote: Fruit op een pizza is als windows op een laptop.
+      description: Middengewicht typeworstelaar, digitaal ongedierteverdelger,
+        zedenkundig computerkraker en frequent jakkenscheerder.
+      links:
+        - icon: github
+          url: https://github.com/rien
+        - icon: mastodon
+          url: https://toot.rxn.be/@rien
+        - url: https://rien.maertens.io
+    - name: Pieter Vander Vennet
+      nickname: pietervdvn
+      photo: https://zeus.ugent.be/zeuswpi/O_nr6ggA.jpg
+      quote: Is dat al gemapped?
+      description: Na de master Wiskundige informatica af te werken waarbij hij enkele jaren in het Zeus-bestuur heeft gezeten, komt hij nu nog graag eens langs om te kelderen of om het over open kaarten en routeplanning te hebben. Daarnaast is hij ook fan van een overdosis hackerspace en lichtjes, waarvoor hij ieder jaar naar Leipzig afzakt en Zeus-mensjes meeneemt daarnaartoe.
+      links:
+        - icon: map
+          url: https://osm.org/user/Pieter%20Vander%20Vennet
+        - icon: github
+          url: https://github.com/pietervdvn
+        - icon: mastodon
+          url: https://en.osm.town/@pietervdvn
+    - name: Robbe Van Herck
+      nickname: Kobbe
+      photo: https://random.robbevanherck.be/static/49.png
```

Comment:
Pls no hotlinking, upload to pics.zeus.gent instead

Reply:
Done @redfast00